motoman_sia5d_moveit_config/launch/moveit_planning_execution.launch

```diff
@@ -0,0 +1,53 @@
+<launch>
+  <!-- The planning and execution components of MoveIt! configured to run -->
+  <!-- using the ROS-Industrial interface. -->
+
+  <!-- Non-standard joint names:
+       - Create a file [robot_moveit_config]/config/joint_names.yaml
+           controller_joint_names: [joint_1, joint_2, ... joint_N]
+       - Update with joint names for your robot (in order expected by rbt controller)
+       - and uncomment the following line: -->
+  <rosparam command="load" file="$(find motoman_sia5d_support)/config/joint_names_sia5d.yaml"/>
+
+  <!-- the "sim" argument controls whether we connect to a Simulated or Real robot -->
+  <!--  - if sim=false, robot_ip and controller arguments are required -->
+  <arg name="sim" default="true" />
+  <arg name="robot_ip" unless="$(arg sim)" doc="IP of controller" />
+  <arg name="controller" unless="$(arg sim)"
+       doc="Series of the controller (dx100, dx200, fs100 or yrc1000)" />
+
+  <!-- load the robot_description parameter before launching ROS-I nodes -->
+  <include file="$(find motoman_sia5d_moveit_config)/launch/planning_context.launch" >
+    <arg name="load_robot_description" value="true" />
+  </include>
+
+  <!-- run the robot simulator and action interface nodes -->
+  <group if="$(arg sim)">
+    <include file="$(find industrial_robot_simulator)/launch/robot_interface_simulator.launch" />
+  </group>
+
+  <!-- run the "real robot" interface nodes -->
+  <!--   - this typically includes: robot_state, motion_interface, and joint_trajectory_action nodes -->
+  <!--   - replace these calls with appropriate robot-specific calls or launch files -->
+  <group unless="$(arg sim)">
+    <node pkg="rosservice" type="rosservice" name="robot_enable" args="call --wait /robot_enable"/>
```

Comment:
@shaun-edwards: is this something we want to do -- automatically -- in a launch file?

I understand that this is convenient, but 'safety' never is convenient.

Reply:
That's a good point. It is probably safer for the user to handle this separately. I can remove the line and create another pull request.